source.Rmd

```diff
@@ -602,16 +627,25 @@ We're going to carry out this hypothesis test with $\alpha = 0.0001$.
 <!-- TODO: Hypothesis test -->
 
 ```{r EDA Diameter_Difference One-way ANOVA, echo=T}
-# TODO: Hypothesis testing (One-way ANOVA)
+
+# Assuming your data frame is named 'df_neo_dia'
+# Make sure 'diameter' and 'neo_type' are the correct column names
+
+# Perform ANOVA
+anova_result <- aov(diameter ~ neo_type, data = df_neo_dia)
+summary(anova_result)
+
 ```
+Given significance level of 0.0001, the p-value (0.00448) is still less than 0.0001. Therefore, reject the null hypothesis.
 
-<!-- TODO: Reject / fail to reject null hypothesis -->
+```{r EDA Diameter_Difference Pair t-test, echo=T}
+pairwise.t.test(df_neo_dia$diameter, df_neo_dia$neo_type, p.adjust.method = "bonferroni")
+```
 
-<!-- TODO: If reject null hypothesis, do pairwise t-test to see which pairs are different -->
 
 ### Verdict
 
-<!-- TODO: What does this mean for the question we asked? -->
+The results of the ANOVA test suggest that there are statistically significant differences in average diameters among the different NEO types (Amors, Apollos, Atens, Atiras). 
```

Comment:
What does this imply for the thing we are testing? Which type of asteroid should we mine?